src/battle-dex-search.ts

```diff
@@ -520,7 +520,7 @@ class DexSearch {
 abstract class BattleTypedSearch<T extends SearchType> {
 	searchType: T;
 	/**
-	 * Dex for the mod/generation to search.
+	 * Dex for the formatType/generation to search.
```

Comment:
```suggestion
	 * Dex for the mod/generation to search.
```

This one actually means "mod", which is why I didn't want you to do the same name as that. :p

Reply:
not sure how this still got changed.... i very intentionally skipped this